test/create_load_test.js

```diff
@@ -7,35 +7,13 @@ var Promise = require('promise');
 var stats   = require("taskcluster-lib-stats");
 var debug   = require('debug')('test:entity:create_load');
 
-suite("Entity (create/load)", function() {
-  var ItemV1;
-  test("ItemV1 = Entity.configure", function() {
-    ItemV1 = subject.configure({
-      version:          1,
-      partitionKey:     subject.keys.StringKey('id'),
-      rowKey:           subject.keys.StringKey('name'),
-      properties: {
-        id:             subject.types.String,
-        name:           subject.types.String,
-        count:          subject.types.Number
-      }
-    });
-  });
-
-  var Item;
-  test("Item = ItemV1.setup", function() {
-    Item = ItemV1.setup({
-      credentials:  helper.cfg.azure,
-      table:        helper.cfg.tableName
-    });
-  });
+var makeTests = function(Item, Item2, drain) {
```

Comment:
Here's what I came up with:
```
helper.contextualSuites("Entity (modify)", [
  {
    context: "Azure",
    options: function() {
      return {
        Item: Item.setup({
          credentials:  helper.cfg.azure,
          table:        helper.cfg.tableName
        })  
      };  
    }   
  }, {
    context: "Azure",
    options: function() {
      return {
        Item: Item.setup({
          inMemory: true,
          table:    'items'
        })  
      };  
    }   
  }
], function(context, options) {
  var Item = options.Item;

  setup(function() {
    return Item.ensureTable();
  }); 

  test("Item.create, Item.modify, Item.load", function() {
    var id = slugid.v4();
```

It basically calls the options function, then calls the suite function with its result..